RevenueCat.xcodeproj/xcshareddata/xcschemes/BackendIntegrationTests.xcscheme

```diff
@@ -62,7 +62,7 @@
          </BuildableReference>
       </BuildableProductRunnable>
       <StoreKitConfigurationFileReference
-         identifier = "../../TestingApps/PurchaseTester/RevenueCat_IntegrationPurchaseTesterConfiguration.storekit">
+         identifier = "../../Tests/TestingApps/PurchaseTester/RevenueCat_IntegrationPurchaseTesterConfiguration.storekit">
```

Comment:
Was this change a side effect from another PR?

Reply:
I should have added a comment but I forgot 😞
This was left over from #1414, it was an outdated reference